decepticonlp/transforms/perturb.py

```diff
@@ -31,25 +31,30 @@ def insert_space(word, ignore=True):
     return word[:index] + " " + word[index:]  # insert space
 
 
-def swap(word, ignore=True):
+def shuffle(word, mid=True, ignore=True):
     """
-    Swaps two adjacent characters in a word which are not at the either end
-    Implies that the word is at least four characters long
-
-    word=input()
-
-    #If input's length is less than 4
-    swap(word)      #Input Hey
-    Assertion Error
-
-    #If input's length is greater than or equal to 4
-    swap(word)      #Input WHAT
+    if mid=True:
+    shuffles the characters of a word at random, barring the initial and last character
+    else:
+    swaps any two characters of a word at random, barring the initial and last character
+    
+    
+    word = "Adversarial"
+    print(shuffle('Adversarial',mid=True))
+    Aaidsvrreal
+    
+    word = "WHAT"
+    print(shuffle('WHAT',mid=False))
     WAHT
     
-    :word: word to be edited
-    :ignore: default (True), boolean if assertions should be ignored
-
-    -returns word with random character swaps
+    :param word : word to be shuffled
+    :param mid : 
+    if set, it shuffle all the characters barring the initial and last
+    if not set, it swap any two characters barring the initial and last 
+                
+    
+    returns shuffled word with first and last character intact
```

Comment:
`nit` 
Two statements on `returns` and `rtype` missing here. This is just for avoiding documenting work later on :)

Reply:
More info here: https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html